ultralytics/solutions/distance_calculation.py

```diff
@@ -39,22 +37,21 @@ def __init__(
         self.centroid_color = centroid_color
 
         # Prediction & tracking information
-        self.clss = None
         self.names = names
         self.boxes = None
         self.line_thickness = line_thickness
         self.trk_ids = None
 
         # Distance calculation information
         self.centroids = []
-        self.pixel_per_meter = pixels_per_meter
 
         # Mouse event information
         self.left_mouse_count = 0
         self.selected_boxes = {}
 
         # Check if environment supports imshow
         self.env_check = check_imshow(warn=True)
+        self.window_name = "Ultralytics Solutions"
 
     def mouse_event_for_distance(self, event, x, y, flags, param):
```

Comment:
Looks like `flags` and `param` parameters aren't being used, could they be removed or do they need to be relocated?

Reply:
@Burhan-Q no, these can not be removed, because it's the arguments for Mouse Event.